src/Example.tsx

```diff
@@ -1,10 +1,17 @@
 import { FC } from 'react';
+import { Button } from '@chakra-ui/react';
 import styled from '@emotion/styled';
+import TimeSelectBottomSheet from 'domain/moit/TimeSelectBottomSheet';
+import { useModal } from 'hooks/useModal';
 
 const Example: FC = () => {
+  const selectDateBottonSheet = useModal();
```

Comment:
P3
```suggestion
  const selectDateBottomSheet = useModal();
```
요 밑에도 다 Botton으로 오타났다 ㅎㅎ 근데 여기 실제 사용부 아니니까 상관 없을듯~
디벨롭에 한치의 실수도 용납하지 않는다면 수정하고 아님 말고 ㅎㅎ

Reply:
새벽코딩 ㅜㅜㅜ 너무 부끄럽습네유 . 오탈자 수정 완료 ~!!